pytmc/bin/pragmalint.py

```diff
@@ -84,6 +87,35 @@ def lint_pragma(pragma):
     match = PRAGMA_RE.match(pragma)
     if not match:
         raise LinterError()
+
+    try:
+        pragma_setting = match.groupdict()["setting"]
+    except KeyError:
+        # if no configuration region in the pragma was detected then fail
+        raise LinterError()
+
+    config_lines = PRAGMA_LINE_RE.findall(pragma_setting)
+    if len(config_lines) == 0:
+        # It is not acceptable to lack configuration lines
+        # At a minimum "pv: " must exist
+        raise LinterError()
```

Comment:
Let's move these comments into the `LinterError` message itself - it'll help the user determine what's wrong with the pragma.